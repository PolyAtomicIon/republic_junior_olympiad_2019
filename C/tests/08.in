64 22
328 493 560 912 408 591 373 982 451 964 305 795 997 732 321 810 401 989 473 888 558 787 990 344 902 944 952 379 822 965 860 554 573 566 927 756 445 356 628 412 400 659 412 385 957 517 352 529 359 876 317 539 543 732 734 418 917 809 990 511 552 771 427 640 