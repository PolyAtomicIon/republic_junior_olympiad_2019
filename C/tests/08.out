328 7709 94390 801833 5309184 29186033 138567307 583656153 223786034 777124830 250870269 808239184 508203688 155497249 328245653 533555027 574297626 56882114 688692757 353857713 217265172 393771251 539921848 78639591 282218578 942278493 575804910 174966455 540095483 100471399 883255546 125716885 972372132 912614040 694289838 547043516 342924233 766515719 335199131 350508393 150885993 954925437 141999132 39570856 789909522 491858289 767809310 51661271 373460390 971827124 999579852 939462744 932276834 666304893 269195216 961224717 647192236 610066385 428790839 406755732 655791623 859372459 553600679 475547343 